modules/enabling-scanner-v4-operator.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-scanner-v4-operator_{context}"]
+= Enabling RHACS Scanner V4 when installing with the Operator
```

Comment:
The procedure in this section describes installing via the Operator's UI, may be ideal to supplement the UI instructions with the yaml instructions.  Something along the lines of:

**Central instructions**

Via Operator UI:

```
<existing procedure>
```

Via Central custom resource yaml, set:
```
scannerV4.scannerComponent: Enabled
```

**Secured Cluster instructions**

Via Operator UI:

```
<existing procedure>
```

Via SecuredCluster custom resource yaml, set:
```
scannerV4.scannerComponent: AutoSense
```